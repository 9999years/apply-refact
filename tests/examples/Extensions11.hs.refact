[("tests/examples/Extensions11.hs:2:1-36: Suggestion: Use newtype instead of data\nFound:\n  data Set (cxt :: * -> *) a = Set [a]\nPerhaps:\n  newtype Set (cxt :: * -> *) a = Set [a]\nNote: decreases laziness\n",[])]
