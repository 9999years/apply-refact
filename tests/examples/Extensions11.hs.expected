{-# LANGUAGE KindSignatures #-}
data Set (cxt :: * -> *) a = Set [a]
